### ISSUE TITLE
Whatsapp and Facebook

### ISSUE THREAD

**Issue by meetgyn** [D3VL/L3MON#134]
** Is your feature request related to a problem? Please describe. **
I miss the option of whatsapp and facebook

** Describe the solution you'd like **
You could have the option to record whatsapp and facebook conversations, this way the application would be more complete.

** Describe alternatives you've considered **
Another valid option that I missed, would be to create a shared login screen. For example: A parent can log in with his account and monitor his children.
A CIO can log in with your account and monitor your employees.

** Additional context **
![image](https://user-images.githubusercontent.com/3695608/90083685-9a279f00-dce9-11ea-8842-4c3c5d194fe9.png)

**Comment by ThePro1ector** [comment]
I can help you with this. Is it just WhatsApp and Instagram you want ?

**Comment by meetgyn** [comment]
Hello, how are you ?
so, i need a system like this that takes information from whatsapp, like audio conversations, geolocation, if possible also telemetry, and recording of ambient sound. That for my company would be the idea.

**Comment by samjor** [comment]
I'm also interested in adding these kinds of improvements to the app. Could you inform me about this service? Is it possible to include this type of display of messages and notifications from Facebook, instagram, telegram or other communication and message platforms?

**Comment by ThePro1ector** [comment]
> Merhaba nasılsın ?
> bu yüzden, sesli konuşmalar, coğrafi konum, mümkünse telemetri ve ortam sesinin kaydı gibi whatsapp'tan bilgi alan böyle bir sisteme ihtiyacım var. Bu benim şirketim için fikir olurdu.
> 
> Örneğin: [https://xnspy.com/buy-now.html ](https://xnspy.com/buy-now.html)

I think I got it. There are many similar products on the market, but you want a unique system? I can only develop for android (no ios) for a special fee. Contact Mail c4-felix@yandex.com 

![Screenshot_2020-08-20 Dashboard](https://user-images.githubusercontent.com/46825437/90700105-a2ee1700-e28d-11ea-861d-f8e267f8a7b8.png)
![Screenshot_2020-08-20 Dashboard(1)(1)](https://user-images.githubusercontent.com/46825437/90700109-a386ad80-e28d-11ea-990d-cb4f96e88891.png)

**Comment by ThePro1ector** [comment]
> Ayrıca uygulamaya bu tür iyileştirmeler eklemekle de ilgileniyorum. Bana bu hizmet hakkında bilgi verir misiniz? Facebook, instagram, telgraf veya diğer iletişim ve mesaj platformlarından gelen bu tür mesaj ve bildirimleri dahil etmek mümkün müdür?

Yes, it is possible, but you need to know exactly what you want.

**Comment by meetgyn** [comment]
this one from the print would already suit me.
